<ui version="4.0" >
 <class>OmgDataFetcherWizardProgressWidgetBase</class>
 <widget class="QWidget" name="OmgDataFetcherWizardProgressWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTextEdit" name="teResults" >
     <property name="font" >
      <font>
       <family>Lucida Grande</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="midLineWidth" >
      <number>1</number>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="alignment" >
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblStatus" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
